opm/parser/eclipse/EclipseState/Eclipse3DProperties.cpp

```diff
@@ -90,15 +87,56 @@ namespace Opm {
                 }
             }
         }
-    }
 
 
+        /*
+          In this codeblock we explicitly set the ACTNUM value to zero
+          for cells with pore volume equal to zero. Observe the
+          following:
+
+             1. The current processing only considers cells with PORV
+                == 0. Processing of the MINPV keyword for cells with
+                small nonzero poervolumes is currently performed along
+                with the Grid processing code.
+
+             2. The PORO or PORV keyword must be explicitly entered
+                in the deck, using getDoubleGridProperty( ) and
+                relying on autocreation will fail - we therefor
+                explicitly check if the PORO or PORV keywords are
+                present in the deck.
+
+             3. The code below will fail hard if it is called with a
+                grid which does not have full cell information.
+        */
+
+        void ACTNUMPostProcessor( std::vector<int>&       values,
+                                  const EclipseGrid*      eclipseGrid,
+                                  GridProperties<double>* doubleGridProperties)
+        {
+            bool hasPORV = doubleGridProperties->hasKeyword( "PORV" ) || doubleGridProperties->hasKeyword( "PORO");
+            if (!hasPORV)
```

Comment:
Then please make it `const`. I'm pretty paranoid when it comes to even local mutable vars simply because then I must always consider a code path where it might change and I need to scan for potential mutation. I consider it good practice.